### DISCUSSION TITLE
broadcast from a linux box

### DISCUSSION THREAD

**Discussion by mat-m** [philippe44/SpotConnect#53] [Q&A] [answered]
Hello,

I have a Linux box plugged into my sound system. It is not seen as a UpnpServer.

My understanding is that I need to start a UpnpServer on that Linuxbox and install SpotConnect on it to be able to broadcast through it, right ?

Thank you

**Comment by philippe44** [comment, marked as answer]
Yes, it would find any UPnP device and morph it into a Spotify Connect item
